apps/docs/pages/guides/platform/org-based-billing.mdx

```diff
@@ -143,60 +92,62 @@ Compute Credits do not apply to other usage fees like egress, database size, ...
+Each plan includes a quota for unified egress, which is applied to your entire organization.
 
-| Usage Item | Old plan (per project)             | New plan (org based)                    |
-| ---------- | ---------------------------------- | --------------------------------------- |
-| Egress     | 4GB - (2GB Database + 2GB Storage) | 5GB across Database + Storage           |
-|            | Max of 2 free projects             | Max of 2 free orgs - 1 database per org |
+| Plan       | Total Egress Quota | Over-Usage   |
+| ---------- | ------------------ | ------------ |
+| Free       | 5 GB               | -            |
+| Pro        | 250 GB             | $0.09 per GB |
+| Team       | 250 GB             | $0.09 per GB |
+| Enterprise | Custom             | Custom       |
 
-Not only are we giving you an extra 1GB of egress for free, now it is unified across your org. This means that if you aren’t using Supabase Storage, you get even more Database Egress.
+### Egress dashboards
 
-### Multiple free plan projects
+You can see a breakdown of the different types of egress on your [organization usage page](https://supabase.com/dashboard/org/_/usage).
 
-We're not changing the number of free projects you can have. Here are the quotas for the free plan:
+<div>
+  <Image
+    alt="Unified Egress"
+    src={{
+      light: '/docs/img/guides/platform/unified-egress--light.png',
+      dark: '/docs/img/guides/platform/unified-egress.png',
+    }}
+  />
+</div>
 
-| Usage Item                | Old plan (per project)             | New plan (org based)                    |
-| ------------------------- | ---------------------------------- | --------------------------------------- |
-| Egress                    | 4GB - (2GB Database + 2GB Storage) | 5GB across Database + Storage           |
-| Database Space            | 500MB                              | 500MB                                   |
-| Storage Space             | 1GB                                | 1GB                                     |
-| Monthly Active Users      | 50,000 MAU                         | 50,000 MAU                              |
-| Edge Function Invocations | 500K                               | 500K                                    |
-| Edge Function Count       | 10                                 | 10                                      |
-| Realtime Message Count    | 2 million                          | 2 million                               |
-| Realtime Peak Connections | 200                                | 200                                     |
-|                           | Max of 2 free projects             | Max of 2 free orgs - 1 database per org |
+## Billing Examples
 
-If you are currently running 2 free projects, **this does require some work from you**. Because we are now working on an Org-level, instead of Projects, you will need to:
+Here are some examples on how the Organization-based Billing affects you.
```

Comment:
**thought:** In the past we had these two terminologies `Project-based` and `Organization-based`. With this PR `Organization-based` was removed in various places. I'm wondering whether it makes sense to not "reintroduce" this term here since we only have one kind of billing now.